src/openshift/component.ts

```diff
@@ -248,6 +248,14 @@ export class Component extends OpenShiftItem {
 
         if (!repoURI) return null;
 
+        const gitRef = await vscode.window.showInputBox({
+            value: 'master',
+            prompt: 'Specify git reference (branch/tags/commits | default: master)',
+            validateInput: (value: string) => {
+                if (!value.trim()) return 'Empty reference, specify master if no reference needed.';
```

Comment:
should accept empty value, which would mean 'master'.

Reply:
Add tests for validation. See example here https://github.com/redhat-developer/vscode-openshift-tools/blob/master/test/openshift/component.test.ts#L212.